### ISSUE TITLE
Default serverless.yml has deprecated content

### ISSUE THREAD

**Issue by jthomerson** [serverless/serverless#2057]
I generated a new 1.0 project today using the quickstart from serverless.com:

```bash
# Install serverless globally
npm install serverless -g
# Create an AWS Lamdba function in Node.js
serverless create --template aws-nodejs
# Deploy to live AWS account
serverless deploy
```

The sample serverless.yml file contained a defaults section. Then when I tried using it, I got the following error:

Serverless: Deprecation Notice: the "defaults" property in serverless.yml is deprecated. The "stage", "region" & "variableSyntax" properties has been moved to the "provider" property instead. Please update your serverless.yml file asap. For more info, you can check our docs.

**Comment by eahefnawy** [comment]
Dang! Nice catch @jthomerson 👍 ... I just PRed updates to the template files. We're now encouraging the `provider` object instead of `defaults`

**Comment by eahefnawy** [comment]
oh apologies! I just noticed that you and @rowanu PRed a fix as well ☺️ . There was also outdated variable reference which is included in my PR.

**Comment by eahefnawy** [comment]
Resolved with PR #2063
